### PR TITLE
Complete incomplete URL during feed import

### PR DESCRIPTION
**Pull request by annando**
This should fix this problem: https://forum.friendi.ca/display/0b6b25a83059e49137ea7f5969152780

### CODE REVIEW

File: include/html2bbcode.php [hunk 1]
```diff
@@ -313,15 +313,69 @@ function ($matches) use (&$codeblocks) {
...
 	$message = preg_replace_callback('#\[codeblock-([0-9]+)\]#iU',
 		function ($matches) use ($codeblocks) {
-            $return = '';
-            if (isset($codeblocks[intval($matches[1])])) {
-                $return = $codeblocks[$matches[1]];
-            }
+	    $return = '';
+	    if (isset($codeblocks[intval($matches[1])])) {
+		$return = $codeblocks[$matches[1]];
+	    }
```
Comment (MrPetovan) [inline include/html2bbcode.php]: There still are indenting spaces in this block of modified code.

File: include/html2bbcode.php [hunk 2]
```diff
@@ -313,15 +313,69 @@ function ($matches) use (&$codeblocks) {
...
+
+/**
+ * @brief Sub function to complete incomplete URL
+ *
+ * @param array $matches Result of preg_replace_callback
+ * @param string $basepath Basepath that is used to complete the URL
+ *
+ * @return string The expanded URL
+ */
+function AddHostnameSub($matches, $basepath) {
```
Comment (MrPetovan) [inline include/html2bbcode.php]: Can you please use `camelCase` for function names?

File: include/html2bbcode.php [hunk 3]
```diff
@@ -313,15 +313,69 @@ function ($matches) use (&$codeblocks) {
...
+
+/**
+ * @brief Complete incomplete URLs in BBCode
+ *
+ * @param string $body Body with URLs
+ * @param string $basepath Basepath that is used to complete the URL
+ *
+ * @return string Body with expanded URLs
+ */
+function AddHostname($body, $basepath) {
```
Comment (MrPetovan) [inline include/html2bbcode.php]: Can you please use `camelCase` for function names?

File: include/html2bbcode.php (hunk 2 above)
Comment (annando) [reply include/html2bbcode.php]: Isn't this?

Comment (MrPetovan) [reply include/html2bbcode.php]: `camelCase` doesn't have an uppercase first letter. `StudlyCaps` or `PascalCase` do.

Comment (annando) [reply include/html2bbcode.php]: This is confusing :-)

**Pull request merged**